controllers/clusteroperator.go

```diff
@@ -114,15 +113,6 @@ func setStatusCondition(conditionType osconfigv1.ClusterStatusConditionType,
 	}
 }
 
-// getOperandVersions returns current version
-func getOperandVersions() []osconfigv1.OperandVersion {
-	operandVersions := []osconfigv1.OperandVersion{}
-	if releaseVersion := os.Getenv("RELEASE_VERSION"); len(releaseVersion) > 0 {
-		operandVersions = append(operandVersions, osconfigv1.OperandVersion{Name: "operator", Version: releaseVersion})
-	}
-	return operandVersions
-}
-
```

Comment:
unused function

Reply:
Is the linter complaining about this?

Reply:
yes, don't worry tho' when we need it, it will be in git history..